src/main/webapp/deletefile.html

```diff
@@ -26,42 +26,36 @@ <h2>Delete Configuration</h2>
       <a href="createfile.html">Create Configuration</a>
     </div><br>
     <div id="delete-search">
-      <form id="delete-form">
-        <span>Customer ID:</span>
-        <input list="customer-ids">
-        <datalist id="customer-ids">
-        </datalist><br>
-        <span>Account ID:</span>
-        <input list="account-ids">
-        <datalist id="account-ids">
-          <option value="Account 1">
-          <option value="Account 2">
-          <option value="Account 3">
-        </datalist><br>
+      <form onSubmit="return deleteVendor(this)">
+      	<p>Enter the Vendor ID of the configuration you want to delete.</p>
+				<input type="text" name="vendorID" size="25" />
```

Comment:
the indention looks sort of off here...looks like github is reading spaces as tabs (?)

Reply:
I believe that the indentation is correct here.

Reply:
sgtm